src/TestHelper/IncludeInTestsAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+[AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly)]
+public abstract class IncludeInTestsAttribute : Attribute, IApplyToContext
+{
+    public void ApplyToContext(TestExecutionContext context)
+    {
+        var currentFilters = Environment.GetEnvironmentVariable("TEST_FILTER")?.Split(new char[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Why even try to split? YAGNI?

Reply:
No, because of the need to run tests locally. The difference with SqlP is that SqlP has no "Default" tests, they are all engine bound. In SC while developing locally what if you wanted to run RabbitMQ and Default tests? It's true that one could set the TEST_FILTER to All.